Applications/Utils/ModelPreparation/PartitionMesh/NodeWiseMeshPartitioner.cpp

```diff
@@ -518,29 +519,6 @@ void NodeWiseMeshPartitioner::writeASCII(const std::string& file_name_base)
     writeNodesASCII(file_name_base);
 }
 
-void NodeWiseMeshPartitioner::resetGlobalNodeIndices()
-{
-    for (std::size_t i = 0; i < _mesh->_nodes.size(); i++)
-    {
-        _mesh->_nodes[i]->setID(_nodes_global_ids[i]);
-    }
-    // sort
-    std::sort(_mesh->_nodes.begin(), _mesh->_nodes.end(),
-              [](const MeshLib::Node* a, const MeshLib::Node* b) {
-                  return a->getID() < b->getID();
-              });
-}
-
-void NodeWiseMeshPartitioner::writeGlobalMeshVTU(
```

Comment:
Sometimes it is useful to have a global mesh for checking the correction of the node indexing after partitioning. It is Ok to remove it, the global mesh will be generated in a post processing.